hathor/builder/sysctl_builder.py

```diff
@@ -27,6 +33,7 @@ def build(self) -> Sysctl:
         root = Sysctl()
         root.put_child('core', HathorManagerSysctl(self.artifacts.manager))
         root.put_child('p2p', ConnectionsManagerSysctl(self.artifacts.p2p_manager))
+        root.put_child('fa', FeatureActivationSysctl(self.artifacts.bit_signaling_service))
```

Comment:
Maybe `core.features.`?

Reply:
Changed in https://github.com/HathorNetwork/hathor-core/pull/962/commits/2fa908fd1cb31ad6f5b0c863d668509f5883b160

Reply:
Thanks!